src/viur/core/modules/file.py

```diff
@@ -757,6 +821,7 @@ def getUploadURL(
         file_skel["pendingparententry"] = db.keyHelper(node, self.addSkel("node").kindName) if node else None
         file_skel["pending"] = True
         file_skel["weak"] = True
+        file_skel["public"] = public
```

Comment:
This was 
```py
        file_skel["public"] = True
```
in the original PR, but I think it's wrong?! @akelch

Reply:
yes, this was wrong, this is how i tested it in the admin